crates/iceberg-catalog/src/service/event_publisher/kafka/mod.rs

```diff
@@ -0,0 +1,79 @@
+pub(crate) mod vendor;
+
+use std::collections::HashMap;
+
+#[cfg(feature = "kafka")]
```

Comment:
you can just put `#[cfg(feature = "kafka")]` on the mod `kafka` in event_publisher instead